doc/nrf/ug_thread_certification.rst

```diff
@@ -73,7 +73,7 @@ Complete the following steps to prepare for the certification tests:
         cd ncs/nrf/samples/openthread/cli/
         west build -b nrf52840dk_nrf52840 -- -DOVERLAY_CONFIG=harness/overlay-cert.conf -DCONFIG_OPENTHREAD_LIBRARY=y
 
-   * If building using Visual Studio Code, you must first `create the application <Creating an application_>`_ using the CLI sample, and then `build the application <Building an application_>`_.
+   * If building using |VSC|, you must first `create the application <Creating an application_>`_ using the CLI sample, and then `build the application <Building an application_>`_.
```

Comment:
```suggestion
   * If building using |nRFVSC|, you must first `create the application <Creating an application_>`_ using the CLI sample, and then `build the application <Building an application_>`_.
```
I would normally agree that |VSC| is fine here, but since we link to the VSC documentation twice in the rest of the sentence, it might be better to use the extension name shortcut here. I'm open to that being challenged though.

Reply:
I would keep this as |VSC| so it matches the heading we use in instructions. But that's more of a gut feeling that based on anything concrete.